sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryParsingErrors.scala

```diff
@@ -542,11 +542,11 @@ private[sql] object QueryParsingErrors extends QueryErrorsBase {
   }
 
   def notAllowedToAddDBPrefixForTempViewError(
-      database: String,
+      viewName: String,
```

Comment:
We should pass the view name as `Seq[String]`, and call `toSQLId` in this method to generate a string

Reply:
Good idea. Done.